src/ui/zcl_abapgit_gui_router.clas.abap

```diff
@@ -681,7 +681,9 @@ CLASS ZCL_ABAPGIT_GUI_ROUTER IMPLEMENTATION.
         TRY.
             " Hide HTML Viewer in dummy screen0 for direct CALL SCREEN to work
             li_html_viewer->set_visiblity( abap_false ).
-            zcl_abapgit_objects=>jump( ls_item ).
+            IF zcl_abapgit_data_utils=>jump( ls_item ) IS INITIAL.
```

Comment:
hmm, just add the IF type = TABU here? instead of having a side effect inside the IF?